### DISCUSSION TITLE
Chitin can be damage which makes it useless as a crafting material

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14584] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Chitin can be damaged by explosions/projectiles/melee.. and cannot be used for crafting if condition <100

![image](https://github.com/user-attachments/assets/78032885-2cb1-48ed-aada-5f66db442dbc)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/6f8143000c9dd594b401c9b69d552e850dd1183f (commit in our private repo), coming up in the next unstable!
